jcl/src/java.base/share/classes/openj9/internal/criu/security/CRIUConfigurator.java

```diff
@@ -101,6 +152,52 @@ public static void setCRIURestoreMode() {
 					System.out.println(provider + " restored.");
 				}
 			}
+			// If any algorithms are available from when we took a checkpoint in our algorithm cache
+			// we can now allow callers to get values from this cache.
+			if (cachedAlgorithms.size() > 0) {
```

Comment:
Suggest `!isEmpty()` instead of `size() > 0` (the former is potentially less expensive).

Reply:
Made use of `!isEmpty()` here as suggested.